calendar.lua

```diff
@@ -41,6 +42,7 @@ if awful.widget.calendar_popup then
     widget:connect_signal(
       "mouse::enter",
       function ()
+        calendar:call_calendar(0, "tr", awful.screen.focused() )
```

Comment:
It'd be nicer if this were

```lua
self:call_calendar(....)
```

Reply:
Also, could you please remove the white space between the last 2 parens?

Reply:
One more thing, since the position `"tr"` is now set by `call_calendar`, it can be removed  from line 40 where the calendar widget is instantiated:
```lua
calendar = awful.widget.calendar_popup.month()
```